components/homme/src/theta-l_kokkos/cxx/HyperviscosityFunctorImpl.cpp

```diff
@@ -276,6 +285,7 @@ void HyperviscosityFunctorImpl::run (const int np1, const Real dt, const Real et
       // Fix w at surface:
       // Adjust w_i at the surface, since velocity has changed
       if (m_process_nh_vars) {
```

Comment:
I think this line is causing the implicit capture of `this`.

Reply:
Thanks -- that was it.